include/SDL_video.h

```diff
@@ -1131,6 +1131,36 @@ extern DECLSPEC void SDLCALL SDL_SetWindowMaximumSize(SDL_Window * window,
 extern DECLSPEC void SDLCALL SDL_GetWindowMaximumSize(SDL_Window * window,
                                                       int *w, int *h);
 
+/**
+ * Get the OS-provided scale factor to apply to the contents of the window.
+ * 
+ * The content scale is how much larger the OS wants us to draw elements
+ * like text to match the user's preferences 
+ * (usually based on DPI of the monitor).
+ * 
+ * This is effectively a more convenient and precise version of doing 
```

Comment:
This comment isn't accurate - it's less precise if it's used for backbuffer-related things unless the same rounding is used in user code as what the OS (or SDL backend code in some cases) does.

Reply:
Good point

Reply:
New comment: 
> SDL_GetWindowSizeInPixels() divided by SDL_GetWindowSize() is approximately equivalent to these values, differing slightly due to rounding behaviors.